src/vmm/src/devices/virtio/vsock/csm/connection.rs

```diff
@@ -98,8 +98,9 @@ use crate::logger::IncMetric;
 
 /// Trait that vsock connection backends need to implement.
 ///
-/// Used as an alias for `Read + ReadVolatile + Write + WriteVolatile + AsRawFd` (sadly, trait
-/// aliases are not supported, https://github.com/rust-lang/rfcs/pull/1733#issuecomment-243840014).
+/// Used as an alias for `Read + ReadVolatile + Write + WriteVolatile + AsRawFd`
```

Comment:
```suggestion
/// Used as an alias for `ReadVolatile + Write + WriteVolatile + AsRawFd`
```
nit: apparently I forgot to update this comment when the `Read` bound became obsolte